client/js/components/pages/TeamsPage.js

```diff
@@ -38,26 +38,25 @@ class TeamsPage extends Component {
     let adminMenu = null;
     if (isAdmin) {
       adminMenu = (
-        <section className="navigation">
-          <Link to="/team/new" className="button">Add team</Link>
-        </section>
+        <div className="btn-group" role="group" aria-label="Actions menu">
+          <Link className="btn btn-secondary btn-sm" to="/team/new">Add team</Link>
```

Comment:
I can see you added aria attributes all over the site + lots of titles/alt messages that I didn't remember to add, which is really good. But don't we need title on links as well?